src/styles/header/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  position: relative;

```

Comment:
why is it here?

Reply:
position relative is for positioning img-logo